local-tests/tests/testUseValidLitActionCodeGeneratedSessionSigsToExecuteJsClaimMultipleKeys.ts

```diff
@@ -0,0 +1,106 @@
+import { LIT_ENDPOINT_VERSION } from '@lit-protocol/constants';
+import { LIT_TESTNET } from 'local-tests/setup/tinny-config';
+import { getLitActionSessionSigs } from 'local-tests/setup/session-sigs/get-lit-action-session-sigs';
+import { TinnyEnvironment } from 'local-tests/setup/tinny-environment';
+
+/**
+ * ## Scenario:
+ * Testing the capability to claim keys using Lit Action PKP session sigs. This test ensures that multiple keys can be claimed correctly.
+ *
+ * - Given: Lit Action PKP sessionSigs are properly generated for the environment.
+ * - When: These sessionSigs are used to execute JS code within Lit Action.
+ * - And: The Lit Action JS code attempts to claim a key using the provided sessionSigs.
+ * - Then: The claim operation should successfully return signatures, derived key IDs, and validate the existence and structure of claimed results.
+ * *
+ * Test Commands:
+ * ❌ Not supported in Cayenne
+ * ❌ Not supported in Manzano
+ * ✅ NETWORK=localchain yarn test:local --filter=testUseValidLitActionCodeGeneratedSessionSigsToExecuteJsClaimMultipleKeys
+ */
+export const testUseValidLitActionCodeGeneratedSessionSigsToExecuteJsClaimMultipleKeys =
+  async (devEnv: TinnyEnvironment) => {
+    devEnv.setUnavailable(LIT_TESTNET.CAYENNE);
+    devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+    const alice = await devEnv.createRandomPerson();
+
+    const litActionSessionSigs = await getLitActionSessionSigs(devEnv, alice);
+
+    const res = await devEnv.litNodeClient.executeJs({
+      sessionSigs: litActionSessionSigs,
+      code: `(async () => {
+      Lit.Actions.claimKey({keyId: "foo"});
+      Lit.Actions.claimKey({keyId: "bar"});
```

Comment:
Isn't there a step required before claiming a PKP? Someone has to make it claimable right or can anyone claim any PKP? @joshLong145

Reply:
@DashKash54 the `keyId` is postfixed with the `ipfs id` so it makes the claim unique to the Lit Action. So you are only claim in context to the lit action as the `app id`

Reply:
Hmm I thought the app had to do something before for making the PKP claimable. Turns out your just need to provide the `keyId`? @joshLong145